views/templates/container/ltiExecutionContainerForm.tpl

```diff
@@ -0,0 +1,15 @@
+<?php
+use oat\tao\helpers\Template;
+?>
+
+<h1>Launching test...</h1>
+
+<form action="<?= get_data('launchUrl') ?>" method="post" id="launch-test-form" style="display:none;"">
+<?php 
+foreach (get_data('launchParams') as $key => $value) {
```

Comment:
is this smarty ?  if yes maybe let's try to use native functions
if not maybe use specific functions build in

Reply:
No template engine, just regular php, but with alternative syntax (said Jérôme).